### DISCUSSION TITLE
Client error message

### DISCUSSION THREAD

**Discussion by elrod68** [abuzuhri/Amazon-SP-API-CSharp#452] [Q&A]
Hello,

Many thanks for creating this. It's extremely helpful and the greatest feature is the throttling handling, it's quite remarkable.
I am facing the following issue:
Migrated our app from MWS API and using ProductPricing.GetCompetitivePricingAsync and GetListingOffersBatchAsync to get competitive prices and lowest offers for a range of SKUs.
Whenever an SKU does not exist in Amazon, we get a ClientError, however we cannot locate the relevant error message. MWS was providing something like
 is an invalid SellerSKU for marketplace
or
SellerSKU maps to an invalid ASIN
Is there something like this available using this API ?
Thanks in advance

**Comment by RenzoF** [comment]
I am not sure if this is a bug on Amazon side, when I test this with an invalid SKU amazon doesn't send any errors all I get is this 
{"payload":[{"SellerSKU":"Y9gvfdgdfgdfg8","status":"ClientError"}]}
Based on the docs I should have an Errors array as well as the payload.

**Reply by elrod68** [reply]
Thanks a lot for the reply.
It seems so, checked the response with isDebug true and it's 
{"statusCode":200,"content":"{\"payload\":[{\"SellerSKU\":\"XXX\",\"status\":\"ClientError\"}]}" ...
where XXX was the requested SKU
